ci/format.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
I may be missing something but I'm not sure this script is needed - does it do anything that `ruff format .` doesn't do? (other than also running `ruff check`, which I'd prefer to be separate really